src/main/java/wooteco/subway/dao/JdbcSectionDao.java

```diff
@@ -36,19 +37,9 @@ public void save(Section section, Long lineId) {
         save(section, lineId, 0);
     }
 
-    private void save(Section section, Long lineId, int index) {
-        Map<String, Object> param = new HashMap<>();
-        param.put("line_id", lineId);
-        param.put("up_station_id", section.getUpStationId());
-        param.put("down_station_id", section.getDownStationId());
-        param.put("distance", section.getDistance());
-        param.put("index_num", index);
-        jdbcInsert.execute(param);
-    }
-
     @Override
     public void save(Sections sections, Long lineId) {
-        LinkedList<Section> values = sections.getSections();
+        List<Section> values = sections.getSections();
```

Comment:
values라는 변수명은 코드를 파악하기 어렵게 만듭니다.
일급 컬렉션 같은 특정 상황이 아니면 의미 있는 이름을 부여해주세요.

```suggestion
        List<Section> sections = sections.getSections();
```